README.md

```diff
@@ -20,6 +20,7 @@ addon | version | summary
 --- | --- | ---
 [account_group_invoice_line](account_group_invoice_line/) | 10.0.1.0.0 | Add option to group invoice lines per account
 [account_invoice_check_total](account_invoice_check_total/) | 10.0.1.0.0 | Check if the verification total is equal to the bill's total
+[account_invoice_refund_option](account_invoice_refund_option/) | 10.0.1.0.0 | Allows you to create directly a refund without starting from an invoice
```

Comment:
This is automatically managed by a bot, so please don't change it (remove last commit), as it will conflict for sure when other PRs are merged.